e2e_tests/tests/test_system.py

```diff
@@ -471,3 +471,11 @@ def test_pytorch_parallel() -> None:
     exp.run_basic_test_with_temp_config(
         config, conf.official_examples_path("trial/mnist_pytorch"), 1
     )
+
+@pytest.mark.e2e_cpu  # type: ignore
+def test_fail_on_first_validation() -> None:
+    exp_id = exp.run_failure_test(
+        conf.fixtures_path("no_op/fail_on_first_validation.yaml"),
+        conf.fixtures_path("no_op"),
+        "failed on first validation",
+    )
```

Comment:
Should we also verify that the trial is in an errored state?  Or is that part of the run_failure_test?

Reply:
it checks this, just a little indirectly - there must be a trial in the 'ERRORED' state with this log line, so the check for the log line will fail if there is no errored trial.